### DISCUSSION TITLE
Renaming Excalidraw Scripts

### DISCUSSION THREAD

**Discussion by HansiMeier** [zsviczian/obsidian-excalidraw-plugin#338] [Script Engine]
Until now I copied manually Excalidraw Scripts into Markdown documents and saved them in the Scripts folder. Naturally I named the markdown documents and tried to keep the names short, expressing clearly the meaning of the command in my words, added a prefix to all Excalidraw Scripts, and used my mother tongue to name them. The new Script store is much more convenient, but if I rename the Scripts for the above reasons I might run into issues with updates. For many Scripts that are now in the library that might not be an issue at all, but for more complex Scripts and Scripts using external connections (i.e. OCR) updates would be essential.

Am I the first one?😆

**Comment by HansiMeier** [comment]
Again me. When installed directly from the script store, then no prefix is necessary. The scripts can be searched in the command pallett with "Script" or "downloaded" (The is the name of the folder in which the scripts are installed from the store.

**Comment by zsviczian** [comment]
I could add a setting where you can define custom names for scripts immediately in the script store, somehow like this:
![image](https://user-images.githubusercontent.com/14358394/148102115-429f4349-313e-4356-bc4f-3bde14414ae2.png)
